src/index.js

```diff
@@ -1,17 +1,16 @@
+// src/index.js
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import createRoot from 'react-dom';
+import { Provider } from 'react-redux';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import store from './redux/store';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+createRoot(
+  <Provider store={store}>
+    <React.StrictMode>
+      <App />
+    </React.StrictMode>
+  </Provider>,
+  document.getElementById('root'),
```

Comment:
- You have done well so far, but I have this error, as shown below, when I try to launch your application which fails to let your component mount. Please fix this. I suggest using the code below:
![fefwef](https://github.com/geekelo/hello-react-front-end/assets/73607512/a9534ba4-2da1-4796-8cdc-491cb70623ac)


```javascript
import ReactDOM from "react-dom/client";
const root = ReactDOM.createRoot(document.getElementById("root"));

root.render(
  <Provider store={store}>
    <React.StrictMode>
      <App />
    </React.StrictMode>
  </Provider>,
);
```